controller/engine_controller.go

```diff
@@ -1512,10 +1512,16 @@ func (ec *EngineController) startRebuilding(e *longhorn.Engine, replica, addr st
 			return
 		}
 
+		fastReplicaRebuild, err := ec.ds.GetSettingAsBool(types.SettingNameFastReplicaRebuildEnabled)
+		if err != nil {
+			log.WithError(err).Errorf("Failed to get %v setting", types.SettingNameFastReplicaRebuildEnabled)
```

Comment:
return here?

Reply:
Yes, should return error. Fixed, thank you.